source/index.html.md

```diff
@@ -23,46 +35,92 @@ Pick which SDK you want to use from the tabs on the right (or in the nav on mobi
 </p>
 
-```objective_c
-# git clone https://github.com/muxinc/stats-sdk-objc.git
+```bitmovin--html
+<!-- Include bitmovin-mux after the core Bitmovin javascript file -->
+<script src="//bitmovin-a.akamaihd.net/bitmovin-player/stable/6/bitmovinplayer.min.js"></script>
+<script src="//src.litix.io/bitmovin/2/bitmovin-mux.js"></script>
+```
+
+<p class="lang-specific bitmovin">
+  Include the bitmovin-mux file after the core Bitmovin javascript file in the page. Use the Mux-hosted version of the script to receive automatic updates (the API will not change within major versions). Note that <code>stable</code> and <code>6</code> in the example on the right can be replaced with the Release Channel and player version (5 or 6), respectively, that you want to use.
+</p>
+
+```ooyala--html
+<!-- Include ooyala-mux after the core Ooyala javascript files -->
+<script src="//player.ooyala.com/static/v4/stable/latest/core.min.js"></script>
+<!-- Insert other Ooyala plugin files here -->
+<script src="//src.litix.io/ooyala/2/ooyala-mux.js"></script>
+```
+
+<p class="lang-specific ooyala">
+  Include the bitmovin-mux file after the core Bitmovin javascript file in the page. Use the Mux-hosted version of the script to receive automatic updates (the API will not change within major versions). Note that <code>stable</code> and <code>6</code> in the example on the right can be replaced with the Release Channel and player version (5 or 6), respectively, that you want to use.
+</p>
 
+```objective-c--shell
+$ git clone https://github.com/muxinc/stats-sdk-objc.git
+```
+```objective-c
 // for iOS
 @import MUXSDKStats;
 
 // for tvOS
 @import MUXSDKStatsTv;
 ```
 
-<p class="lang-specific objective_c">
+<p class="lang-specific objective-c">
   Include the correct Mux Objective-C SDK for your project by cloning our repository and dragging the framework into your Xcode project. The <code>Frameworks</code> folder contains two folders, one for iOS and one for tvOS. Inside these folders, there are 3 additional folders containing different architecture combinations. The <code>fat</code> folder contains a library with all architectures in one. This library cannot be used when compiling for submission to the App Store as it contains the simulator architectures that are not used by any Apple devices. You can use the framework in the <code>release</code> folder when building a release version of your application, or you can run <a href="https://gist.github.com/brett-stover-hs/b25947a125ff7e38e7ca#file-frameworks_blogpost_removal_script_a-sh">a script to strip unneeded architectures</a>. Finally, don't forget to add the correct import statement for your target platform.
```

Comment:
We probably should, but right now it's beta. I'd almost rather mark it as beta for now, and then when we pull it out of beta, we need to have a release process (and document how people get the latest). Thoughts?